meilisearch/src/routes/logs.rs

```diff
@@ -0,0 +1,282 @@
+use std::convert::Infallible;
+use std::io::Write;
+use std::ops::ControlFlow;
+use std::pin::Pin;
+use std::str::FromStr;
+use std::sync::Arc;
+
+use actix_web::web::{Bytes, Data};
+use actix_web::{web, HttpResponse};
+use deserr::actix_web::AwebJson;
+use deserr::{DeserializeError, Deserr, ErrorKind, MergeWithError, ValuePointerRef};
+use futures_util::Stream;
+use index_scheduler::IndexScheduler;
+use meilisearch_types::deserr::DeserrJsonError;
+use meilisearch_types::error::deserr_codes::*;
+use meilisearch_types::error::{Code, ResponseError};
+use tokio::sync::mpsc;
+use tracing_subscriber::filter::Targets;
+use tracing_subscriber::Layer;
+
+use crate::error::MeilisearchHttpError;
+use crate::extractors::authentication::policies::*;
+use crate::extractors::authentication::GuardedData;
+use crate::extractors::sequential_extractor::SeqHandler;
+use crate::LogRouteHandle;
+
+pub fn configure(cfg: &mut web::ServiceConfig) {
+    cfg.service(
+        web::resource("stream")
+            .route(web::post().to(SeqHandler(get_logs)))
+            .route(web::delete().to(SeqHandler(cancel_logs))),
+    );
+}
+
+#[derive(Debug, Default, Clone, Copy, Deserr, PartialEq, Eq)]
+#[deserr(rename_all = lowercase)]
+pub enum LogMode {
+    #[default]
+    Fmt,
```

Comment:
Maybe for later: "fmt" looks too implementation-oriented, I think the name should reflect the intended purpose of the log.

So here "Human" or "HumanReadable" or "Pretty" or "Print", something like that (in decreasing order of preference)

Reply:
Yeah, tbh I was also a bit against fmt but had no idea oh what to write and was finding `human readable` too long to write.
Didn’t think of `human` though 😅 
That looks good; if you’re ok I would like to do it in this PR so you and Kero don’t have to update your code again once the PR lands

Reply:
The benchmark isn't using the human mode, so go ahead :-)

Reply:
it doesn't really convey the "activity" and is subjective

Reply:
Yeah, I'm more on Louis's side on this one. I went with human